pack.yaml

```diff
@@ -8,7 +8,7 @@ keywords:
   - mice
   - ipam
   - dns
-version: 1.0.2
+version: 1.0.3
```

Comment:
deserves a bigger version bump

Reply:
the workflow fails. Nothing has been claimed yet so no remediation is needed